### PR TITLE
Remove CentOS8 GitHub actions CI Tests

### PR DESCRIPTION
**Pull request by liangwen12year**
As CentOS8 is already EOL, remove CentOS8 CI integration tests.

Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

**Review by tyll — APPROVED** [review]
thank you! :+1:

**Comment by liangwen12year** [comment]
Hi Till, I changed the PR slightly and may need your review again, thanks.

**Review by ffmancera — DISMISSED** [review]

**Comment by ffmancera** [comment]
Thanks!

**Review by tyll — DISMISSED** [review]
:+1:

File: .github/run_test.sh [hunk 1]
```diff
@@ -31,8 +30,8 @@ EOF
 EXEC_PATH=$(dirname "$(realpath "$0")")
 PROJECT_PATH=$(dirname "$(realpath "$EXEC_PATH../")")
 
-# Default to CentOS 8
-OS_TYPE=c8
+# Default to CentOS Stream 8
```
Comment (tyll) [inline .github/run_test.sh:33]: To avoid having to update this in the future, this could be changed to:
```suggestion
# Default
```

**Review by tyll — APPROVED** [review]

**Review by tyll — APPROVED** [review]

**Pull request merged**